Gemfile

```diff
@@ -114,7 +114,7 @@ group :development do
   gem "brakeman", "~> 4.10", require: false # Brakeman detects security vulnerabilities in Ruby on Rails applications via static analysis
   gem "bundler-audit", "~> 0.7" # bundler-audit provides patch-level verification for Bundled apps
   gem "derailed_benchmarks", "~> 1.8", require: false # A series of things you can use to benchmark a Rails or Ruby app
-  gem "erb_lint", "~> 0.0.34", require: false # ERB Linter tool
+  gem "erb_lint", github: "splitwise/erb-lint", branch: "jess/remove-version-pinning", require: false # ERB Linter tool
```

Comment:
see the PR description here

Reply:
I'm somewhat concerned about pointing to a fork's branch. The past two times we needed to do something like this we would fork it ourselves to ensure that the branch won't be altered or deleted. What do you think?

Reply:
Makes perfect sense, i'll fork it and update it while waiting for `erb_lint` to release the new version

Reply:
I forked over `thepracticaldev/erb_lint` because I don't have permission to fork over `forem/*`